src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeProcessTests.cs

```diff
@@ -252,6 +251,96 @@ public async Task Dispose_ProcessAlreadyKilled_NoException()
             testSubject.Process.Should().BeNull();
         }
 
+        [TestMethod]
+        public void IsRunning_ServerWasNeverStarted_False()
+        {
+            var testSubject = CreateTestSubject();
+
+            testSubject.IsRunning.Should().BeFalse();
+        }
+
+        [TestMethod]
+        public void IsRunning_ServerWasNeverStartedAndWasDisposed_False()
+        {
+            var testSubject = CreateTestSubject();
+
+            testSubject.Dispose();
+
+            testSubject.IsRunning.Should().BeFalse();
+        }
+
+        [TestMethod]
+        public async Task IsRunning_ServerWasStarted_True()
+        {
+            var fakeNodeExePath = CreateScriptThatPrintsPortNumber(123, isHanging: true);
+            var startupScriptPath = "dummy path";
+
+            var nodeLocator = SetupNodeLocator(fakeNodeExePath);
+            var testSubject = CreateTestSubject(startupScriptPath, nodeLocator: nodeLocator.Object);
+
+            Process spawnedProcess = null;
+            try
+            {
+                await testSubject.Start();
+                spawnedProcess = testSubject.Process;
+
+                testSubject.IsRunning.Should().BeTrue();
+            }
+            finally
+            {
+                SafeKillProcess(spawnedProcess);
+            }
+        }
+
+        [TestMethod]
+        public async Task IsRunning_ServerWasStartedThenDisposed_False()
+        {
+            var fakeNodeExePath = CreateScriptThatPrintsPortNumber(123, isHanging: true);
+            var startupScriptPath = "dummy path";
+
+            var nodeLocator = SetupNodeLocator(fakeNodeExePath);
+            var testSubject = CreateTestSubject(startupScriptPath, nodeLocator: nodeLocator.Object);
+
+            await testSubject.Start();
+
+            testSubject.IsRunning.Should().BeTrue();
+
+            testSubject.Dispose();
+
+            testSubject.IsRunning.Should().BeFalse();
+        }
+
+        [TestMethod]
+        public async Task IsRunning_ServerWasRestarted_True()
```

Comment:
This test checks everything that is tested in the two previous tests, so the other tests don't add much value. It might be clearer to have one test appropriately named and commented.

Reply:
I removed IsRunning_ServerWasStartedThenDisposed_False and IsRunning_ServerWasStarted_True.
The other two (IsRunning_ServerWasNeverStarted_False and IsRunning_ServerWasNeverStartedAndWasDisposed_False) test a bit different things.